livelossplot/main_logger.py

```diff
@@ -24,10 +24,11 @@ def __init__(
         current_step: int = -1,
         auto_generate_groups_if_not_available: bool = True,
         auto_generate_metric_to_name: bool = True,
-        group_patterns: List[Tuple[str, str]] = [
+        group_patterns: List[Tuple[Pattern, str]] = [
             (r'^(?!val(_|-))(.*)', 'training '),
```

Comment:
Here we shouldn't have space after (e.g. `'training'` not `'training '`).

Reply:
done

Reply:
done